src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -43,8 +47,11 @@ export default function WebexAvatar({
   style,
 }) {
   const {avatar, displayName, status} = usePerson(personID);
+  const [randomColor] = useState(avatarColors[Math.floor(Math.random() * avatarColors.length)]);
   const type = displayStatus ? status : null;
-  const cssClasses = webexComponentClasses('avatar', className);
+  const avatarInitials = displayName && displayName.split(' ').map((name) => name.charAt(0)).slice(0, 2);
+  const avatarColorClassName = !avatar && displayName && `avatar-color-${randomColor}`;
+  const cssClasses = webexComponentClasses(`avatar ${avatarColorClassName}`, className);
```

Comment:
look at webexComponentClasses, it has a separate argument for non-prefixed optional classes (the 4th, I think).